### PR TITLE
Fix CI, part 2

### PR DESCRIPTION
**Pull request by sobolevn**
Refs https://github.com/typeddjango/django-stubs/pull/1108

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def find_stub_files(name: str) -> List[str]:
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.930",
+    "mypy>=0.930,<0.970",  # TODO: remove after CI is fixed
```
Comment (andersk) [inline setup.py:23]: This shouldn’t be necessary for the sake of CI. CI does `pip install -r ./requirements.txt`, and `requirements.txt` already includes `mypy==0.961` (and also `-e .[compatible-mypy]` which constrains `mypy` to `<0.970`).

Comment (sobolevn) [reply setup.py:23]: Any ideas how to fix it?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          pip install -U pip setuptools wheel
+          pip install -U pip 'setuptools<64' wheel
```
Comment (andersk) [inline .github/workflows/test.yml:24]: I think the two other copies of this need to be patched too?

**Comment by andersk** [comment]
I found two ways to get it working again: #1122 or #1123.

**Pull request closed without merging**